src/lib.rs

```diff
@@ -164,6 +166,7 @@ impl<'a, Writer: Write> Printer<'a, Writer> {
         show_position_panel: bool,
         border_style: BorderStyle,
         use_squeeze: bool,
+        columns: u8,
```

Comment:
This is a breaking change for `hexyl`-as-a-library. This whole `new` method is really unfortunate. There are too many arguments even before this PR. We should really use a builder-pattern instead.

But that's not really the problem of your PR.

Reply:
I would be willing to work on implementing a builder pattern if you're committed to pushing a breaking change anyway.